org.freedesktop.Sdk.Extension.tup.yaml

```diff
@@ -0,0 +1,28 @@
+id: org.freedesktop.Sdk.Extension.tup
```

Comment:
what is the usecase here? are there any apps using it or want to use it?

Reply:
I use it for [this game](https://github.com/nmlgc/ssg/tree/linux) which I'm going to submit once this PR is merged or closed, and possibly more in the future.